include/cantera/numerics/Integrator.h

```diff
@@ -175,11 +175,6 @@ class Integrator
         warn("setMethodType");
     }
 
-    //! Set the linear iterator.
-    virtual void setIterator(IterType t) {
-        warn("setInterator");
-    }
-
```

Comment:
While it's not likely that this function is used directly by any external users, I don't think the `setIterator` function should be removed without first being deprecated for one release. That is, for now it should have an `@deprecated` message added to the docstring, and a call to `warn_deprecated` in the function body (for all implementations of the function). See for example the [`absmax`](https://github.com/Cantera/cantera/blob/master/include/cantera/base/utilities.h#L178) function, which will be removed after Cantera 2.5.

Reply:
Am I correct in assuming that 
https://github.com/Cantera/cantera/blob/26e47e59fe36465cf3fbc988077650efd8f46804/include/cantera/numerics/CVodesIntegrator.h#L54-L56
still should be removed on deprecation?

Reply:
Yes, that's correct -- it is safe to remove the implementation from the derived class in this case.